### PR TITLE
add elad to CODEOWNERS

### PR DESCRIPTION
**Pull request by eladkal**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -65,9 +65,12 @@
 /airflow/providers/google/ @turbaszek
 /airflow/providers/snowflake/ @turbaszek @potiuk @mik-laj
 /airflow/providers/cncf/kubernetes @jedcunningham
+/docs/apache-airflow-providers-cncf-kubernetes @jedcunningham
```
Comment (uranusjr) [inline .github/CODEOWNERS]: I think this is supposed to be the last line of this section

Comment (eladkal) [reply .github/CODEOWNERS]: I was wondering if we have a way to catch docs, code and tests together rather than seperated line for each

Comment (potiuk) [reply .github/CODEOWNERS]: I think not. Look here https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

Comment (potiuk) [reply .github/CODEOWNERS]: Glob (used there) is rather limited - we would have to rename the docs to have the same structure as providers to get a 'sane' way of specifying it (which might be a good idea btw before we split providers).

Comment (eladkal) [reply .github/CODEOWNERS]: :(

**Review by potiuk — APPROVED** [review]

**Pull request merged**